docs/testing.md

```diff
@@ -6,26 +6,26 @@ next: docs/logging.md
 
 We highly recommend working in the style of [test-driven development](http://agiledata.org/essays/tdd.html) when creating probot apps. It is frustrating to constantly create real GitHub events in order to test an app. Redelivering webhooks is possible and can be accessed in your app's [settings](https://github.com/settings/apps) page under the **Advanced** tab. We do offer the above documented `simulate` method to help make this easier; however, by writing your tests first, you can avoid repeatedly recreating actual events from GitHub to check if your code is working.
 
-For our testing examples, we use [jest](https://facebook.github.io/jest/), but there are other options that can perform similar operations. Here's an example of creating a robot instance and mocking out the GitHub API:
+For our testing examples, we use [jest](https://facebook.github.io/jest/), but there are other options that can perform similar operations. Here's an example of creating an app instance and mocking out the GitHub API:
 
 ```js
-// Requiring probot allows us to mock out a robot instance
-const {createRobot} = require('probot')
+// Requiring Probot allows us to get an `Application` instance
+const {Application} = require('probot')
```

Comment:
~Variables starting with an uppercase are typically reserved for classes;~ I've just realized that we are now exporting the class, not an instance of it; that's a major breaking change, and I think we should be careful about it. Maybe having that change as a separate PR?

Reply:
^ This also crossed my mind, but keep in mind this PR is against the typescript branch